app/controllers/bids_controller.rb

```diff
@@ -32,14 +32,14 @@ def new
 
   def confirm
     @auction = AuctionViewModel.new(current_user, Auction.find(params[:auction_id]))
-    @bid = BidPresenter.new(PlaceBid.new(params, current_user).dry_run)
+    @bid = BidPresenter.new(PlaceBid.new(params, current_user, via).dry_run)
```

Comment:
woooo! thoughts on named args for `BidPresenter` so we can be less dependent on arg order?